runner/src/client/sass/application.scss

```diff
@@ -67,11 +67,17 @@ $govuk-global-styles: true;
     &--hidden-titles {
       @extend .govuk-visually-hidden;
     }
-
   }
   &__key {
     &--hidden-titles {
       width: 100%;
     }
   }
 }
+
+h4,
+h5,
+h6 {
+  margin-top: 0;
+  margin-bottom: 10px;
+}
```

Comment:
Little worried about this having unintended side effects since it's applying to all `h4, h5, h6` afaik

Reply:
Made this unique instead